curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bcc9c4a6dd6d7dd3e6357.md

```diff
@@ -10,6 +10,26 @@ dashedName: what-is-the-css-box-model
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What is the CSS box model, and how does it work?
+
+The CSS box model is a fundamental concept for web development. It defines how HTML elements are structured and positioned. If you understand this model you will be able to controll the size, spacing and appearance of the elements on your website.
+
+In the CSS box model, every element is sorrounded by a box. This box consists of four elements: the content area, the padding, the border, and the margin. 
+
+The content area is the innermost part of the box. It's the space that contains the actual content of an element like text or images. You can see the content area in this representation of an HTML element in the CSS box model.
+
+The padding is the area immediately after the content area. It's the space between the content area and the border of an element. With the padding you can add space around the content to improve it's readability. You can set different values for the top, right, bottom and left padding with the padding property. This is an example with the padding shorthand property where we set the top margin to fifteen pixels, the left margin to five pixels, the bottom margin to two pixels and the left margin to eight pixels.
+
+The border is the outer edge or outline of an element in the CSS box model. It's the visual boundary of the element. You can customize the border style, width, color and other properties using the border property. Here's an example where we set the border to a width of five pixels, a solid state and a color of blue. If you omit a value the default property of that value will be used, that's medium for the width, none for the style and the current color for the color. 
```

Comment:
```suggestion
The border is the outer edge or outline of an element in the CSS box model. It's the visual boundary of the element. You can customize the border style, width, color and other properties using the `border` property. Here's an example where we set the border to a width of five pixels, a solid state and a color of blue. If you omit a value the default property of that value will be used, that's medium for the width, none for the style and the current color for the color. 
```

Reply:
this paragraph mentions "here is an example" but no example is shown. so maybe we can just show a css example below it so the text makes sense since we are not using images